app/timetables/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from app.timetables.models import Timetable
+
```

Comment:
I was reading two scoops of django today and it advises we use explicit relative imports when importing from another module in the current app.
So instead of from app.timetables.models import Timetable let's use from .models import Timetable since models and admin sit under the same sub-app `timetables`
Also, add from __future__ import absolute_import to the top of the file